TaylorIntegrationTests/ReporterComparator.swift

```diff
@@ -33,14 +33,14 @@ func !=(lhs: [String:AnyObject], rhs: [String:AnyObject]) -> Bool {
 
 class ReporterComparator {
     
-    func compareReporters(_ firstReporterPath: String, secondReporterPath: String) -> Bool {
-        var violations1 = JSONToViolationParser().parseFile(firstReporterPath)
+    func compareReporter(atPath path1: String, withReporterAtPath path2: String) -> Bool {
```

Comment:
This is the function that gets two paths, then parses this files to get violations, then compares each violation through iteration. 
All this is happening in `compareReporter(atPath:withReporterAtPath:)` function of `ReporterComparator` class.
The Reporter word leads to a delusion here(we also have reporters in this project).
Can you think about a better name, like `compareViolations`, or for a better place/implementation of this functionality(extension)?